src/modules/notebook-panel/notebookPanel.styles.js

```diff
@@ -6,7 +6,7 @@ const notebookStyles = StyleSheet.create({
   panel: {
     width: 400,
     height: '100%',
-    backgroundColor: themes.PRIMARY_BACKGROUND_COLOR,
+    backgroundColor: '#ffffff',
```

Comment:
@sdamaraju We don't want to hard code colors. They should be grabbed from themes. In this case it would be SECONDARY_BACKGROUND_COLOR.